src/opnsense/mvc/app/models/OPNsense/IPsec/Swanctl.xml

```diff
@@ -44,9 +44,25 @@
                 <local_addrs type=".\IKEAdressField">
                     <Required>N</Required>
                 </local_addrs>
+                <local_port type="OptionField">
+                    <Required>N</Required>
+                    <Default></Default>
+                    <OptionValues>
+                        <port500 value="">500 (default)</port500>
+                        <port4500 value="4500">4500 (NAT-T)</port4500>
+                    </OptionValues>
+                </local_port>
                 <remote_addrs type=".\IKEAdressField">
                     <Required>N</Required>
                 </remote_addrs>
+                <remote_port type="OptionField">
+                    <Required>N</Required>
+                    <Default></Default>
```

Comment:
same here

Reply:
I removed the default and tested it. Still works as expected. Thank you.